### PR TITLE
Added analogsombraVia is-a-dev-cli

### PR DESCRIPTION
**Pull request by analog-sombra**
Added analogsombravia the CLI

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
404 and please provide an email.

**Comment by analog-sombra** [comment]
@WilliamDavidHarrison  email address is added, please kindly check it again sir.

**Review by wdhdev — CHANGES_REQUESTED** [review]
I can't reach your website

**Comment by analog-sombra** [comment]
@WilliamDavidHarrison  https://analog-sombra.github.io/MyPortfolio

**Review by omsenjalia — DISMISSED** [review]
👍👍

File: domains/analogsombra.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "analog-sombra",
+        "email": "analogsombra@gmail.com"
+      },
+      "record": {
+        "CNAME": "analogsombra.dev"
```
Comment (WilliamDavidHarrison) [inline domains/analogsombra.json:8]: I can't reach this address though. If you want to use GitHub Pages please change this to `analog-sombra.github.io`.
```suggestion
        "CNAME": "analog-sombra.github.io"
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Review by wdhdev — APPROVED** [review]

**Pull request merged**